lib/pages/main/main_vc.dart

```diff
@@ -1,456 +1,282 @@
+            _buildCustomTile(3, Icons.science, 'guide_tab', context),
+            _buildCustomTile(4, Icons.settings, 'settings_title', context),
+            _buildCustomTile(5, Icons.info, 'info_tab', context),
+            SizedBox(
+              height: 60,
+            ),
+            Center(
+              child: Column(
+                mainAxisSize: MainAxisSize.max,
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  Text(packageInfo != null
+                    ? 'version ${packageInfo.version} (build ${packageInfo.buildNumber})'
+                    : '',
+                    style: TextStyle(
+                      color: Colors.grey,
+                      fontSize: 8.0
                     ),
                   ),
-                );
-              },
-            )),
-        Positioned.fill(
-          child: StreamBuilder<bool>(
-            stream: loadingStream.stream,
-            initialData: true,
-            builder: (BuildContext ctxt, AsyncSnapshot<bool> snapshot) {
-              if (snapshot.hasData == false || snapshot.data == false) {
-                return Container();
-              }
-              return Utils.loading(
-                snapshot.data,
-              );
-            },
-          ),
-        )
-      ],
+                ],
+              ),
+            ),
+          ],
+        ),
+      ),
     );
   }
 
-  _createBiteReport() async {
-    var createReport = await Utils.createNewReport('bite');
-    loadingStream.add(false);
-    if (createReport) {
-      await Navigator.push(
-        context,
-        MaterialPageRoute(builder: (context) => BitingReportPage()),
-      );
-    } else {
-      print('Bite report was not created');
-      loadingStream.add(false);
-      await Utils.showAlert(MyLocalizations.of(context, 'app_name'),
-          MyLocalizations.of(context, 'server_down'), context);
-    }
-  }
+  Widget _uuidWithClipboard(){
+    return FutureBuilder(
+      future: UserManager.getUUID(),
+      builder: (BuildContext context, AsyncSnapshot<String?> snapshot) {
+        if (snapshot.connectionState == ConnectionState.waiting) {
+          return CircularProgressIndicator();
+        } else if (snapshot.hasError) {
+          return Text('Error: ${snapshot.error}');
+        }
 
-  _createAdultReport() async {
-    var createReport = await Utils.createNewReport('adult');
-    loadingStream.add(false);
-    if (createReport) {
-      await Navigator.push(
-        context,
-        MaterialPageRoute(builder: (context) => AdultReportPage()),
-      );
-    } else {
-      print('Adult report was not created');
-      loadingStream.add(false);
-      await Utils.showAlert(MyLocalizations.of(context, 'app_name'),
-          MyLocalizations.of(context, 'server_down'), context);
-    }
+        print(snapshot.data);
+        return Row(
+          children: [
```

Comment:
Could you add the string 'ID: ' before pasting the user uuid? (sorry, this was not in the screenshots I share with you)

Example: 'ID: 7a5c1cf0-7fe6-4a8b-81ba-67e72475742b'

'ID: ' -> color black
'7a5c1cf0-7fe6-4a8b-81ba-67e72475742b' -> color grey (whatever if it looks lighter color than 'ID')

Reply:
Sure, the "copy to clipboard" also in black or grey?

![image](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/assets/157690872/e46d48b7-37ec-47f7-a67d-b0b5dcf21340)

Reply:
Black is perfect :+1: But it grey works better on your opinion, go for it